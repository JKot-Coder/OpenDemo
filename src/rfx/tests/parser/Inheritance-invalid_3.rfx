// :PARSER

Bar: {
    Name: "Unknown",
    Valid: true,
},

Foo: <Bar> false
