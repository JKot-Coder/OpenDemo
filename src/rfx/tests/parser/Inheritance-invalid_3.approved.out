Diagnostic:[
    ../src/rfx/tests/parser/Inheritance-invalid_3.rfx(8): error 20016: The value 'Foo' is of type type 'Bool', but only objects can have parents.
    Foo: <Bar> false
          ^~~
]
